<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefaultCriterionWdgt</class>
 <widget class="QWidget" name="DefaultCriterionWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>427</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Activate cards from these card types:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="active_or_forbidden">
       <item>
        <property name="text">
         <string>having any of these tags:</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>not having any of these tags:</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTreeWidget" name="card_types_tree">
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTreeWidget" name="tags_tree">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>active_or_forbidden</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DefaultCriterionWdgt</receiver>
   <slot>criterion_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>card_types_tree</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>DefaultCriterionWdgt</receiver>
   <slot>criterion_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tags_tree</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>DefaultCriterionWdgt</receiver>
   <slot>criterion_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
